### ISSUE TITLE
Uses an earlier papis release, not the latest 

### ISSUE THREAD

**Issue by raspharaoh** [papis/papis-zotero#54]
The install uninstalls the latest papis 0.14 and installs an earlier release 0.13.

```bash
pip install papis-zotero
Collecting papis-zotero
  Downloading papis-zotero-0.1.2.tar.gz (16 kB)
...
  Attempting uninstall: papis
    Found existing installation: papis 0.14
    Uninstalling papis-0.14:
      Successfully uninstalled papis-0.14
Successfully installed arxiv2bib-1.0.8 chardet-5.2.0 papis-0.13 papis-zotero-0.1.2
```

**Comment by alexfikl** [comment]
Yeah, `papis-zotero` has not yet been updated to the new version and is pinned to the previous release (since they're not particularly compatible).

We should get to it in the coming weeks.

**Comment by alexfikl** [comment]
#55 bumps the required version and should work just fine with Papis 0.14, if you can install from git.

We'll try to make a release soon as well.

**Comment by alejandrogallo** [comment]
I'm not sure why the pypi release did not work here, it seems that the `Publish a Github Release` step got jumped over ?

**Comment by alexfikl** [comment]
@alejandrogallo The release still needs some additional work, I think:
1. Add a `pypi` environment in the `Settings > Environments`.
2. Add a trusted publisher on PyPI for this repository.

Besides that, this is all set up like `papis`, so not sure what other reason it may have to not work..

**Comment by raspharaoh** [comment]
Just installed ver. 0.2. All good.
